src/elements/index.md

```diff
@@ -0,0 +1,114 @@
+</p>
+
+DataJoint Elements provides an efficient approach for neuroscience labs to create and
+manage _scientific data workflows_: the complex multi-step methods for data collection,
+preparation, processing, analysis, and modeling that researchers must perform in the
+course of an experimental study. Elements are a collection of curated modules for
+assembling workflows for several modalities of neurophysiology experiments
+and are designed for ease of integration into diverse custom workflows. This work is
+derived from the developments in leading neuroscience projects and uses the
+[DataJoint API](/core) for defining, deploying, and sharing their data workflows.
+
+An overview of the principles of DataJoint workflows and the goals of DataJoint
+Elements are described in the position paper
+["DataJoint Elements: Data Workflows for Neurophysiology"](https://www.biorxiv.org/content/10.1101/2021.03.30.437358v2).
+
+Below are the projects that make up the family of open-source DataJoint Elements:
+
+<div class="grid cards" markdown>
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Calcium Imaging**
+
+    ---
+
+    A data pipeline for calcium imaging microscopy.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-calcium-imaging/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Array Electrophysiology**
+
+    ---
+
+    A data pipeline for Neuropixels probes.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-array-ephys/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Electrode Localization**
+
+    ---
+
+    A data pipeline for electrode localization of Neuropixels probes.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-electrode-localization/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Miniscope**
+
+    ---
+
+    A data pipeline for miniscope calcium imaging.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-miniscope/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element DeepLabCut**
+
+    ---
+
+    A data pipeline for pose estimation with DeepLabCut.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-deeplabcut/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Facemap**
+
+    ---
+
+    A data pipeline for pose estimation with Facemap.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-facemap/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Visual Stimulus**
+
+    ---
+
+    A data pipeline for visual stimulation with Psychtoolbox.
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/elements/element-visual-stimulus/)
+
+-   :fontawesome-brands-python:{ .lg .middle } **Element Lab**
```

Comment:
@kabilar Sorry - realized later. Does fontawesome have a matlab?
```suggestion
-   :fontawesome-brands-matlab:{ .lg .middle } **Element Lab**
```

Reply:
Looks like they haven't added it yet...
- https://github.com/FortAwesome/Font-Awesome/issues/10999
- Not even on their leaderboard https://fontawesome.com/community/leaderboard/brands

I've been using the Java one since MATLAB is written in Java.

There may be a way to customize but might be a bit more work than needed if we can *live* with it.

https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

Reply:
Thanks team.  I am okay with the Java icon for now.  I have updated the `element-visual-stimulus` icon to Java.